src/assets/stack-blitz/package.json

```diff
@@ -1,40 +1,42 @@
 {
-  "name": "material-aio-example",
+  "name": "example-app",
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^13.0.0-next.0",
-    "@angular/cdk": "^13.0.0-next.0",
-    "@angular/common": "^13.0.0-next.0",
-    "@angular/compiler": "^13.0.0-next.0",
-    "@angular/core": "^13.0.0-next.0",
-    "@angular/forms": "^13.0.0-next.0",
-    "@angular/material": "^13.0.0-next.0",
-    "@angular/material-moment-adapter": "^13.0.0-next.0",
-    "@angular/platform-browser": "^13.0.0-next.0",
-    "@angular/platform-browser-dynamic": "^13.0.0-next.0",
-    "@angular/router": "^13.0.0-next.0",
-    "moment": "^2.29.1",
-    "rxjs": "^6.6.7",
-    "tslib": "^2.2.0",
-    "zone.js": "^0.11.4"
+    "@angular/animations": "13.0.0-rc.0",
+    "@angular/cdk": "${version}",
+    "@angular/common": "13.0.0-rc.0",
+    "@angular/compiler": "13.0.0-rc.0",
+    "@angular/core": "13.0.0-rc.0",
+    "@angular/forms": "13.0.0-rc.0",
+    "@angular/material": "${version}",
+    "@angular/platform-browser": "13.0.0-rc.0",
+    "@angular/platform-browser-dynamic": "13.0.0-rc.0",
+    "@angular/router": "13.0.0-rc.0",
+    "rxjs": "~6.6.0",
+    "sass": "1.35.1",
```

Comment:
I'm curious why this isn't in `devDependencies`?

Reply:
Not sure where this came from. I think it was a leftover from my old webcontainer VM. I'll re-generate the project with rc.2 where this can be removed completely.